sinks/cortex/cortex.go

```diff
@@ -41,15 +43,28 @@ type CortexMetricSink struct {
 	name          string
 	tags          map[string]string
 	traceClient   *trace.Client
+	addHeaders    map[string]string
+	basicAuth     *BasicAuthType
+}
+
+type BasicAuthType struct {
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
```

Comment:
This probably should be of type `util.StringSecret` so it doesn't get printed to console.